iphone/Classes/KrollContext.h

```diff
@@ -175,3 +183,29 @@ KrollContext *GetKrollContext(TiContextRef context);
 @property (nonatomic, readwrite, retain) id invocationArg4;
 
 @end
+
+#if defined(USE_JSCORE_FRAMEWORK) && !defined(TI_USE_KROLL_THREAD)
+
+/**
+ * Handles creating and clearing timers when running with JavaScriptCore
+ */
+@interface JSTimerManager : NSObject
+
+/**
+ * Map of timer identifiers and the underlying native NSTimer.
+ */
+@property (nonatomic, strong) NSMutableDictionary<NSNumber *, NSTimer *> *timers;
```

Comment:
If timers are supposed to be unique identified, an `NSMutableSet` may be more appropriate.

Reply:
Since i need to access timers by their id in various places a set is somewhat cumbersome to work with in that use case.